shell/platform/darwin/ios/ios_gl_context.mm

```diff
@@ -193,7 +193,7 @@
   bool rebind_color_buffer = false;
 
   if (depthbuffer_ != GL_NONE || stencilbuffer_ != GL_NONE ||
-      depth_stencil_packed_buffer_ != GL_NONE) {
+      depth_stencil_packed_buffer_ != GL_NONE || colorbuffer_ != GL_NONE) {
```

Comment:
The following calls will only return the dimensions of the bound render buffer. This happens to be the color buffer. So the other checks are not necessary.

Reply:
Since we are not going to use the buffers anyway, this class is too advanced for our use. For simplicity, I suggest getting rid of the ability to configure the additional buffers.

Reply:
Yep will do a cleanup run in a followup patch that blows the rest away.